rootfs/rootfs/usr/local/bin/forgiving-getty

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -e
+
+usage() {
+	self="$(basename "$0")"
+	cat <<-EOUSAGE
+		usage: $self <tty>
+		   ie: $self ttyS0
+		       $self ttyS1
+	EOUSAGE
+}
+
+tty="$1"
+if ! shift || [ -z "$tty" ]; then
+	usage >&2
+	exit 1
+fi
+
+# https://github.com/systemd/systemd/blob/6a47fd894d601f7e8e88dec4cb35dfb7d7c15eff/src/getty-generator/getty-generator.c#L94-L116
+verify_tty() {
```

Comment:
I feel like this could use at least a bit more explanation (don't need to go overboard since it doesn't get touched too much) -- I'm not sure what 42 refers to below for instance